src/sched/lthread.c

```diff
@@ -46,8 +46,8 @@
 #include "enclave/lthread_int.h"
 #include "enclave/sgxlkl_t.h"
 #include "enclave/ticketlock.h"
-#include "shared/tree.h"
 #include "openenclave/corelibc/oemalloc.h"
+#include "shared/tree.h"
```

Comment:
I snuck them in because there were few changes, and this makes the formatting of lthread.c consistent again.

Can we enforce correct formatting on commit? We keep diverging for files, which causes these problems. @letmaik has already added the formatting commit hooks, so I think it's just a matter of turning them on.